src/main/kotlin/com/bookmarks/models/Price.kt

```diff
@@ -0,0 +1,13 @@
+package com.bookmarks.models
+
+data class Price(val dollars: Int, val cents: Int) : Comparable<Price> {
+    override operator fun compareTo(other: Price): Int =
+        toCents().compareTo(other.toCents())
+
+    operator fun times(discount: Double): Price = ((toCents() * discount).toInt().fromCents())
+
+    companion object {
+        fun Price.toCents(): Int = dollars * 100 + cents
```

Comment:
Why do you place it inside the companion object? :)

Reply:
We want to make function `toCents` next to `fromCents`

Reply:
Why not to move both outside the `companion object`?

Reply:
Fixed.